app/assets/stylesheets/govuk_publishing_components/components/_column-list.scss

```diff
@@ -0,0 +1,20 @@
+.gem-c-column-list {
+  columns: 1;
+
+  @include govuk-media-query($from: tablet) {
+    columns: 2;
```

Comment:
Is that what the data.gov.uk homepage does? Because it seems like gov.uk gives 3 columns to tablet without a problem and we probably don't want to change that.

Reply:
data.gov.uk is 3 columns all the way down, however this, in my opinion, was an oversight on my part when I built that implementation. I disagree that 3 columns on mobile isn't a problem because, whilst it just about looks ok on data.gov.uk, it's a bit of a squeeze. If that content ever gets longer it may become unreadable. On the gov.uk homepage it becomes a single column on mobile so I'll get rid of this column 2 on tablet setup.